### PR TITLE
raft_cmdpb.proto:remove uuid

### PR DESCRIPTION
**Pull request by AndreMouche**
Hi, 
     This PR removed uuid in raft_cmdpb.proto.
     @BusyJay @siddontang @javaforfun PTAL

### CODE REVIEW

**Comment by javaforfun** [comment]
LGTM

File: proto/raft_cmdpb.proto [hunk 1]
```diff
@@ -214,15 +214,13 @@ message RaftRequestHeader {
     // true for read linearization
     optional bool read_quorum                   = 3;
     // 16 bytes, to distinguish request.
```
Comment (javaforfun) [inline proto/raft_cmdpb.proto]: remove comment?

**Comment by BusyJay** [comment]
LGTM

**Comment by siddontang** [comment]
any update @AndreMouche

**Pull request closed without merging**